.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -105,6 +99,7 @@ jobs:
           RELEASE: ${{ if inputs.stage == 'release' && 'yes' || 'no' }}
 
       - name: store-artifact ${{ matrix.asset }}
+        if: ${{ matrix.stage != 'release' && (matrix.component == 'agent' || matrix.component == 'coco-guest-components' || matrix.component == 'pause-image') }}
```

Comment:
This bug'ed me!

I think you meant:

`matrix.stage != 'release' || (matrix.component != 'agent' && matrix.component != 'coco-guest-components' && matrix.component != 'pause-image')`

Reply:
If I'm correct here then it needs to adjust the next store-artifacts's ifs.

Reply:
Yes - you are correct, I think I should have had a `!` before `(matrix.component == 'agent' || matrix.component == 'coco-guest-components' || matrix.component == 'pause-image')`, but your code works too. Thanks for the good spot!

Reply:
Ok, I've updated the logic on all the build jobs to actually work now. Thanks!